components/automate-ui/src/app/pages/project/list/project-list.component.ts

```diff
@@ -145,6 +147,16 @@ export class ProjectListComponent implements OnInit, OnDestroy {
     return '';
   }
 
+  updateButtonLabel(): string {
+    if (this.applyRulesInProgress) {
+      return `Updating Projects ${Math.round(this.percentageComplete * 100)}%...`;
+    }
+    if (this.projectsHaveStagedChanges) {
+      return 'Update Projects';
+    }
+    return 'Projects Up-to-date';
```

Comment:
This looks like it should be capitalized all the way through to me 🙃 `Projects Up-to-Date`

Reply:
i wasnt sure... sounds good to me though, thanks @blakestier!